milestone34.cabal

```diff
@@ -0,0 +1,45 @@
+cabal-version:   3.0
+name:            milestone34
```

Comment:
I've tried to name it `milestone-3-4` but got cabal parsing error.

Reply:
The exact name doesn't matter very much - this will work OK.